express-static/index.js

```diff
@@ -0,0 +1,23 @@
+const express = require('express');
+const app = express();
+const path = require('path');
+const fs = require('fs');
+
+app.get('/image1.jpg', (req, res) => {
+  const pathToImg = path.join(pathToPublic, 'image1.jpg');
+  fs.readFile(pathToImg, (err, data) => {
+    if (err) {
+      res.status(404).send('not found');
+    } else {
+      res.end(data);
+
+    }
+
+  });
+});
+
+const pathToPublic = path.join(__dirname, 'public');
+console.log(pathToPublic);
+const expressStatic = express.static('public');
```

Comment:
Use your joined path here instead of just `public`. All file system operations are relative to `process.cwd()` unless they are given an absolute path.